events-application-rest-api/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -65,4 +65,8 @@ databaseChangeLog:
               - column:
                   name: date
                   type: timestamp
-
+              - column:
```

Comment:
You should create a new `changeSet` for every change to the database. This will create conflicts and **break our database**. Please move your changes to a new changeset.